apps/gitness/package.json

```diff
@@ -32,14 +33,15 @@
     "pluralize": "^8.0.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-hook-form": "^7.53.0",
```

Comment:
Don't see this package used in `apps/gitness`. Are we using it?

Reply:
Resolved.